uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/payment/adyen/AdyenPaymentPresenter.kt

```diff
@@ -134,30 +150,37 @@ class AdyenPaymentPresenter(view: PaymentDropInContract.Actions,
     }
 
     private fun createCardConfig(context: Context, publicKey: String): CardConfiguration {
+        val saveCard = !KarhooUISDKConfigurationProvider.isGuest()
+
         return CardConfiguration.Builder(context, publicKey)
-                .setShopperLocale(Locale.getDefault())
-                .setHolderNameRequire(true)
-                .setShowStorePaymentField(false)
-                .build()
+            .setShopperLocale(Locale.getDefault())
+            .setHolderNameRequired(true)
+            .setShowStorePaymentField(saveCard)
+            .build()
     }
 
     private fun getPaymentMethods(locale: Locale?) {
-        val amount = AdyenAmount(quote?.price?.currencyCode ?: DEFAULT_CURRENCY
-                                 , quote?.price?.highPrice.orZero())
+        val amount = AdyenAmount(
+            quote?.price?.currencyCode ?: DEFAULT_CURRENCY, quote?.price?.highPrice.orZero()
```

Comment:
please put `quote?.price?.highPrice.orZero()` on the next row for proper formatting.